src/model/wallet.js

```diff
@@ -203,7 +204,15 @@ const saveWalletDetails = async (seedPhrase, privateKey, address) => {
     authenticationType: AUTHENTICATION_TYPE.DEVICE_PASSCODE_OR_BIOMETRICS,
   });
   let accessControlOptions = {};
+
+  let isSimulator = false;
+
   if (canAuthenticate) {
+    // eslint-disable-next-line no-undef
+    isSimulator = __DEV__ && (await DeviceInfo.isSimulator());
```

Comment:
@brunobar79 should we prefer __DEV__ over the process.env.NODE_ENV === 'development' checks? (we currently use the latter in App.js)